velox/type/Type.h

```diff
@@ -1269,6 +1269,8 @@ class DateType : public IntegerType {
 
   std::string toString(int32_t days) const;
 
+  std::string toISO8601String(int32_t days) const;
```

Comment:
Make this static. Rename to toIso8601. Add documentation and unit tests.

Reply:
Done.